test/yesql_test.exs

```diff
@@ -128,6 +241,67 @@ defmodule YesqlTest do
     end
   end
 
+  describe "defqueries/2" do
```

Comment:
These are duplicated in full from `defquery` to make sure I was covering the same circumstances.